package.json

```diff
@@ -66,6 +66,7 @@
     "eslint": "^2.8.0",
     "fb-watchman": "^1.9.0",
     "glob": "^7.0.3",
+    "lodash": "4.13.1",
```

Comment:
That's interesting. I've recently had a discussion about a very similar package organization in material-ui and expressed my opinion that it just shouldn't be done because npm doesn't truly support it. They support one main in a package. And the "scoped package" feature falls short in not supporting a central initialization for the whole scope.